api/src/main/java/org/openmrs/module/muzimafingerPrint/api/db/hibernate/HibernateMuzimaFingerprintDAO.java

```diff
@@ -43,8 +42,8 @@ public MuzimaFingerprint findById(Integer id) {
     }
 
     private Session session() {
-        factory.getCurrentSession().setCacheMode(CacheMode.IGNORE);
-        return factory.getCurrentSession();
+        sessionFactory.getCurrentSession().setCacheMode(CacheMode.IGNORE);
```

Comment:
My preference would be to use the getter for the session factory here since you have the getter for it. It's just my preference, but this code is fine.